docs/static/releasenotes.asciidoc

```diff
@@ -24,6 +25,108 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-6-0]]
+=== Logstash 7.6.0 Release Notes
+
+* Feature: Introduce deprecation logger https://github.com/elastic/logstash/pull/11260[#11260]
```

Comment:
```suggestion
* Feature: Introduce deprecation logger for internal classes and plugins.  https://github.com/elastic/logstash/pull/11260[#11260]
```

Reply:
I noticed that #11486 (Docs related to deprecation logger) didn't get picked up.  We could include that PR here rather than making a separate entry.  Something like: 

```
* Feature: Introduce deprecation logger for internal classes and plugins.  https://github.com/elastic/logstash/pull/11260[#11260] and https://github.com/elastic/logstash/pull/11486[#11486]
```
Another option.  We could say "Deprecations are noted in the `log/logstash-deprecation.log` file.